### DISCUSSION TITLE
Export to CSV to be in the timezone of the dashboard settings

### DISCUSSION THREAD

**Discussion by DigNZ** [grafana/grafana#38127] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
When choosing inspect -> data -> Download CSV the time stamps are shown in the database timezone (in this case UTC) not the dashboard timezone. 
There is this pull request from an older version of Grafana:
https://github.com/grafana/grafana/pull/18002
But I can't find equivilent functionality in the latest version.

**Why is this needed**:
This means that the CSV data does not match the data you are viewing. This is not useful for users that do not understand or care to convert between timezones and it should match expectations when download or have options to manage the settings.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by DigNZ** [comment]
I can't find any further update on this - looks like it was how things used to work but no longer do?

Does anyone have any advice on how to export to CSV in the current dashboard timezone settings instead of the database settings?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
